chart/env/dev.yaml

```diff
@@ -103,7 +103,7 @@ mongodbMigration:
 # --- jobs (post-upgrade hooks) ---
 
 cacheMaintenance:
-  action: "backfill"
+  action: "skip"
```

Comment:
This action does not exist, right? It's OK, because it's explicit that the job will not do anything, but I'm afraid that it will exist with error code 1, ie. it will make the whole "sync" being reported as failed in ArgoCD

<img width="391" alt="Capture d’écran 2023-04-06 à 20 08 12" src="https://user-images.githubusercontent.com/1676121/230460267-77978e58-019b-46af-920b-2c69bd0471b4.png">

Reply:
No, it will print
`        logging.warning(f"Wrong action mode selected, supported actions are {supported_actions}.")`

Reply:
Oh, right.